airflow/models/dag.py

```diff
@@ -2532,18 +2532,28 @@ def create_dagrun(
 
     @classmethod
     @provide_session
-    def bulk_sync_to_db(cls, dags: Collection["DAG"], session=NEW_SESSION):
+    def bulk_sync_to_db(
+        cls,
+        dags: Collection["DAG"],
+        processor_subdir: Optional[str] = None,
+        session=NEW_SESSION,
+    ):
         """This method is deprecated in favor of bulk_write_to_db"""
         warnings.warn(
             "This method is deprecated and will be removed in a future version. Please use bulk_write_to_db",
             RemovedInAirflow3Warning,
             stacklevel=2,
         )
-        return cls.bulk_write_to_db(dags, session)
+        return cls.bulk_write_to_db(dags, processor_subdir, session)
```

Comment:
Don't _need_ to/shouldn't change this method, as it's a deprecated alias.

Reply:
reverted